frontend/src/api/helpers.js

```diff
@@ -0,0 +1,14 @@
+function updateOptions(options) {
+  const update = { ...options };
+  if (localStorage.getItem('token')) {
+    update.headers = {
+      ...update.headers,
+      Authorization: `Bearer ${localStorage.getItem('token')}`,
+    };
+  }
```

Comment:
I believe something like this is more readable and avoids the double call to `localStorage`.
```js
const update = { headers: {}, ...options };
const token = localStorage.getItem('token');
if (token) update.headers.Authorization = `Bearer ${token}`;
```
The first line is updated since `headers` might not always exist.